app/controllers/api/account.php

```diff
@@ -3876,6 +3975,99 @@
         $dbForProject->deleteDocument('authenticators', $authenticator->getId());
         $dbForProject->purgeCachedDocument('users', $user->getId());
 
+        // Send email to user informing that MFA is disabled
+        $subject = $locale->getText("emails.mfaRemoved.subject");
+        $customTemplate = $project->getAttribute('templates', [])['email.mfaRemoved-' . $locale->default] ?? [];
+
+        // Appwrite theming for console
+        if ($project->getInternalId() === 'console') {
+            $message = Template::fromFile(__DIR__ . '/../../config/locale/templates/email-base-styled.tpl');
+            $body = Template::fromFile(__DIR__ . '/../../config/locale/templates/email-mfa-altered.tpl');
+
+            $body
+                ->setParam('{{buttonText}}', $locale->getText("emails.mfaRemoved.buttonText"))
+                ->setParam('{{redirect}}', 'http://' . App::getEnv('_APP_DOMAIN') . '/console/account')
```

Comment:
We typically use the hostname from the request instead of the `_APP_DOMAIN` env var. For example:

https://github.com/appwrite/appwrite/blob/90bc303c7cbb683c92ae499a54954cea5a4b43fc/app/controllers/api/account.php#L1011

Should we do the same here?